x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/management/trial_license_complete_tier/install_prebuilt_rules_with_historical_versions.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from 'expect';
+import { FtrProviderContext } from '../../../../../../ftr_provider_context';
+import {
+  deleteAllRules,
+  deleteAllTimelines,
+  deleteAllPrebuiltRuleAssets,
+  createRuleAssetSavedObject,
+  installPrebuiltRulesAndTimelines,
+  deleteRule,
+  getPrebuiltRulesAndTimelinesStatus,
+  createHistoricalPrebuiltRuleAssetSavedObjects,
+  getPrebuiltRulesStatus,
+  installPrebuiltRules,
+  getInstalledRules,
+} from '../../../../utils';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('@ess @serverless @skipInQA install prebuilt rules from package w/ historical versions with mock rule assets', () => {
```

Comment:
`w/` -> `with`

Two characters difference and much more clarity. Same with `w/o` -> `without`